web/Dockerfile

```diff
@@ -19,11 +19,16 @@ WORKDIR /app
 
 ENV NODE_ENV production
 ENV NUXT_TELEMETRY_DISABLED 1
+# Currnently, HOST_ENV is used to decide whether to import the TEMPLATE files of the k8s resources.
+# ./components/lib/templates/*.yaml
+ENV HOST_ENV production
```

Comment:
See also `web/nuxt.config.js`. 
Is there a better way? (These TEMPLATEs files are only needed at build time. Also, these env values must not be present at the application startups. )